common/lib/xmodule/xmodule/capa_module.py

```diff
@@ -172,6 +174,13 @@ def non_editable_metadata_fields(self):
         ])
         return non_editable_fields
 
+    @property
+    def problem_types(self):
```

Comment:
@cahrens there's `@registry.register` all over the `common/lib/capa/capa/responsetypes.py` and it just builds up a dictionary in `common/lib/capa/capa/registry.py`